eq_translations/survey_schema.py

```diff
@@ -292,6 +277,37 @@ def translate(self, schema_translation, language_code):
 
         return SurveySchema(translated_schema)
 
+    def _handle_translation_for_pointer(
```

Comment:
I can't tell from the method name here what's it's meant to do.

Reply:
Feel free to suggest alternatives.

Reply:
It's translating in place right? Which seems a bit dirty to me given it mutates the schema to do so. Either way translate_pointer would be a better name.

Reply:
This will change in the next commit.